src/riak_cs_console.erl

```diff
@@ -100,10 +102,104 @@ cleanup_orphan_multipart(Timestamp) when is_binary(Timestamp) ->
     _ = io:format("~nAll old unaborted orphan multipart uploads have been deleted.~n", []).
 
 
+%% @doc Resolve siblings as quick operation. Assuming login via
+%% `riak-cs attach` RawKey and RawBucket should be provided via Riak
+%% log. Example usage is:
+%%
+%% > {ok, Pid} = riakc_pb_socket:start_link(Host, Port).
+%% > riak_cs_console:resolve_siblings(Pid, <<...>>, <<...>>).
+%% > riakc_pb_socket:stop(Pid).
+%%
+%% or, for CS Bucket and Key name:
+%%
+%% > {ok, Pid} = riakc_pb_socket:start_link(Host, Port).
+%% > RawBucket = riak_cs_utils:to_bucket_name(objects, CSBucketName),
+%% > riak_cs_console:resolve_siblings(Pid, RawBucket, CSKey).
+%% > riakc_pb_socket:stop(Pid).
+%%
+-spec resolve_siblings(pid(), RawBucket::binary(), RawKey::binary()) -> ok | {error, term()}.
+resolve_siblings(Pid, RawBucket, RawKey) ->
+    GetOptions = [{r, all}],
+    PutOptions = [{w, all}],
+    resolve_siblings(Pid, RawBucket, RawKey,
+                     GetOptions, ?DEFAULT_RIAK_TIMEOUT,
+                     PutOptions, ?DEFAULT_RIAK_TIMEOUT).
+
+resolve_siblings(Pid, RawBucket, RawKey, GetOptions, GetTimeout, PutOptions, PutTimeout)
+  when is_integer(GetTimeout) andalso is_integer(PutTimeout) ->
+    io:format("~p: getting ~p:~p~n", [erlang:now(), RawBucket, RawKey]),
+    case riakc_pb_socket:get(Pid, RawBucket, RawKey, GetOptions, GetTimeout) of
+        {ok, RiakObj} ->
+            io:format("~p: Trying to resolve ~p sibling(s) of ~p:~p~n",
+                      [erlang:now(), riakc_obj:value_count(RiakObj), RawBucket, RawKey]),
+            _ = lager:info("Trying to resolve ~p sibling(s) of ~p:~p",
+                           [riakc_obj:value_count(RiakObj), RawBucket, RawKey]),
+            case resolve_ro_siblings(RiakObj, RawBucket, RawKey) of
+                {ok, RiakObj2} ->
+                    R = riakc_pb_socket:put(Pid, RiakObj2, PutOptions, PutTimeout),
+                    io:format("~p: Resolution result: ~p~n", [erlang:now(), R]);
+                {error, R} = E ->
+                    io:format("~p: Not updating ~p:~p: ~p~n",
+                              [erlang:now(), RawBucket, RawKey, R]),
```

Comment:
This log line is to indicate error occurrence. Is it better to include the word "error" (or "ERROR" or "Error ...) for grep'pability?